include/RED4ext/Scripting/Natives/entAnimationControllerComponent.hpp

```diff
@@ -0,0 +1,98 @@
+#include <cstdint>
+
+namespace RED4ext
+{
+namespace anim
+{
+struct ActionAnimDatabase;
+}
+namespace ent
+{
+struct AnimationControlBinding;
+struct AnimationControllerComponent;
+
+struct IKTargetController
+{
+    struct IKTargetData
+    {
+        anim::IKTargetRef ikTargetref;                    // 00
+        Handle<IPositionProvider> positionProvider;       // 10
+        Handle<IOrientationProvider> orientationProvider; // 20
+    };
+    RED4EXT_ASSERT_SIZE(IKTargetData, 0x30);
+
+    AnimationControllerComponent& animationControllerComponent; // 00
+    uint64_t unk08;                                             // 08
+    DynArray<IKTargetData> IKTargetData;                        // 10
+    Handle<anim::IKTargetParams_Update> IKparams;               // 20
+};
+RED4EXT_ASSERT_SIZE(IKTargetController, 0x30);
+
+struct LookAtController
+{
+    struct LookatData
+    {
+        anim::LookAtRef lookAtRef;                  // 00
+        Handle<IPositionProvider> positionProvider; // 10
+        uint32_t unk20;                             // 20
+    };
+    RED4EXT_ASSERT_SIZE(LookatData, 0x28);
+    struct AdditionalLookatData
+    {
+        CName partName;                       // 00
+        DynArray<anim::LookAtRef> lookAtRefs; // 08
+    };
+    RED4EXT_ASSERT_SIZE(AdditionalLookatData, 0x28);
+
+    AnimationControllerComponent& animationControllerComponent; // 00
+    uint32_t unkCounter;                                        // 08
+    void* unk10;                                                // 10
+    bool usingTPPCamera;                                        // 18
+    DynArray<LookatData> lookAtData;                            // 20
+    DynArray<AdditionalLookatData> additionalLookatdata;        // 30
+    Handle<anim::LookAtParams_UpdatePositions> lookAtparams;    // 40
+    uint64_t unk50[5];                                          // 50
+    float unk78;                                                // 78
+    uint32_t unk7C;                                             // 7C
+    uint64_t unk80;                                             // 80
+};
+RED4EXT_ASSERT_SIZE(LookAtController, 0x88);
+
+struct AnimationControllerComponent : ent::IComponent
+{
+    static constexpr const char* NAME = "entAnimationControllerComponent";
+    static constexpr const char* ALIAS = "AnimationControllerComponent";
+
+    uint64_t unk90[7];                                   // 90
+    void* unkC8;                                         // C8
+    void* unkD0;                                         // D0
+    DynArray<uint64_t> unkD8Array;                       // D8
+    DynArray<uint64_t> unkE8Array;                       // E8
+    Ref<anim::ActionAnimDatabase> actionAnimDatabaseRef; // F8
+    anim::AnimDatabaseCollection animDatabaseCollection; // 110
+    LookAtController lookAtController;                   // 120
+    IKTargetController ikTargetController;               // 1A0
```

Comment:
If `LookAtController` is `0x88` bytes, then `0x120 + 0x88 = 0x1A8`.
Please add `RED4EXT_ASSERT_OFFSET` for all named fields of `AnimationControllerComponent` and some fields of other structs.